src/plugins/stake-plugins/types.ts

```diff
@@ -41,8 +41,9 @@ export interface StakePolicy {
 // -----------------------------------------------------------------------------
 // Change Quote
 // -----------------------------------------------------------------------------
+export type ChangeQuoteAction = 'stake' | 'unstake' | 'claim'
```

Comment:
Keep it inline until it needs to be factored out.

Reply:
done